onebusaway-android/build.gradle

```diff
@@ -27,11 +27,14 @@ apply plugin: 'com.android.application'
 
 repositories {
     jcenter()
-    mavenLocal()
     maven {
         // OBA Releases - for comparator to sort alphanumeric routes
         url "http://nexus.onebusaway.org/nexus/content/repositories/releases"
```

Comment:
I'd prefer not to - we've added 3rd party artifacts to the OBA nexus repo in the past, and its come back to bite us (https://github.com/OneBusAway/onebusaway-application-modules/issues/103).  I don't know why Amazon isn't hosting this on JCenter or Maven Central - I asked when I first set this up and they didn't know why either.  I'll check in with them again to see if they've made progress.